examples/amp-story/link-occupies-full-screen.html

```diff
@@ -0,0 +1,65 @@
+<!doctype html>
```

Comment:
If you want to check in your example, do you think you could update the one you already created in your previous PR? We're trying to not check in all the examples we use to develop daily, otherwise we'd already have several hundreds of documents :((

Reply:
Sure! I changed the story to be a single one for these link issues (`invalid-link-tests.html`) and then put tests for both bugs into it.